### PR TITLE
improve docs for akka-http #20699

### PR DESCRIPTION
**Pull request by note**
Made imports in docs snippets more complete and a few minor improvements

Related to https://github.com/akka/akka/issues/20699

### CODE REVIEW

**Comment by ktoso** [comment]
OK TO TEST

File: akka-docs/rst/scala/http/introduction.rst [hunk 1]
```diff
@@ -25,9 +25,8 @@ Akka HTTP was designed specifically as “not-a-framework”, not because we don
 
 Using Akka HTTP
 ---------------
-Akka HTTP is provided in separate jar files, to use it make sure to include the following dependencies::
+Akka HTTP is provided in a separate jar file, to use it make sure to include the following dependency::
 
-  "com.typesafe.akka" %% "akka-http-core" % "@version@" @crossString@
   "com.typesafe.akka" %% "akka-http-experimental" % "@version@" @crossString@
```
Comment (ktoso) [inline akka-docs/rst/scala/http/introduction.rst]: Not quite sure about this change... We do want to explain that it's 2 modules.

How about explaining here that: hey, you should likely use `"com.typesafe.akka" %% "akka-http-experimental" % "@version@" @crossString@` however you should know that the core is actually in a separate module?

**Comment by ktoso** [comment]
LGTM besides https://github.com/akka/akka/pull/20722/files#r65899632
Let's address that and merge.

Thanks a lot @note !

File: akka-docs/rst/scala/http/introduction.rst (hunk 1 above)
Comment (note) [inline akka-docs/rst/scala/http/introduction.rst]: See your point. I added some explanation in the following paragrapth. Does it state clearly what you meant?

**Comment by note** [comment]
I addressed your comment. Also I changed sth in `WebSocketClientExampleSpec.scala` and `WebSocketClientExampleTest.java` because I saw some complaints on gitter.

File: akka-docs/rst/scala/http/introduction.rst (hunk 1 above)
Comment (ktoso) [inline akka-docs/rst/scala/http/introduction.rst]: Yes, very nicely worded - thanks!

File: akka-docs/rst/scala/code/docs/http/scaladsl/WebSocketClientExampleSpec.scala [hunk 2]
```diff
@@ -23,59 +20,60 @@ class WebSocketClientExampleSpec extends WordSpec with Matchers with CompileOnly
...
-    // in a real application you would not side effect here
-    // and handle errors more carefully
-    connected.onComplete(println)
-    closed.foreach(_ => println("closed"))
-
     //#single-WebSocket-request
   }
 
   "half-closed-WebSocket-closing-example" in compileOnlySpec {
+    import akka.actor.ActorSystem
```
Comment (ktoso) [inline akka-docs/rst/scala/code/docs/http/scaladsl/WebSocketClientExampleSpec.scala]: thanks :) that one's from the chat today I think :)

**Comment by ktoso** [comment]
LGTM, let's merge if validation is happy. 
Thanks for the PR @note !

**Comment by ktoso** [comment]
Thanks!

File: akka-docs/rst/scala/code/docs/http/scaladsl/WebSocketClientExampleSpec.scala (hunk 2 above)
Comment (note) [inline akka-docs/rst/scala/code/docs/http/scaladsl/WebSocketClientExampleSpec.scala]: Exactly

**Pull request merged**